docs/pages/management/operations/tls-routing.mdx

```diff
@@ -149,6 +140,18 @@ proxy_service:
 Config version `v2` will prevent other listeners from being created unless
 they are explicitly set in the proxy service configuration.
 
+## Migrating from dual load balancer setups
```

Comment:
The "dual load balancer setup" is quite vague. Additionally in TLS Routing mode we are forwarding by design all the traffic to the  web_proxy_addr so there is no any additional steps needed to be preformed between TLS Routing migration and L7 LB TLS Routing migration.